controllers/kafkaacl_controller.go

```diff
@@ -140,12 +131,16 @@ func (h KafkaACLHandler) get(avn *aiven.Client, i client.Object) (*corev1.Secret
 		return nil, err
 	}
 
-	aivenACL, err := h.getAivenACL(avn, acl)
+	id, err := h.getID(avn, acl)
 	if err != nil {
 		return nil, err
 	}
 
-	acl.Status.ID = aivenACL.ID
+	// todo: remove in v1
+	if acl.Status.ID == "" {
```

Comment:
Why do we need this if? Can we just have `acl.Status.ID = id`?

Reply:
Just want make it explicit.

Reply:
Replaced the whole block btw.